tests/cases/nested_6/config.yml

```diff
@@ -0,0 +1,4 @@
+jtype: MyExecutor
```

Comment:
The test purpose is to test `py_modules`, can use the [simplest case](https://github.com/jina-ai/executor-normalizer/tree/main/tests/cases/executor_4) instead of nested.